lib/archethic_web/controllers/faucet_controller.ex

```diff
@@ -40,8 +42,12 @@ defmodule ArchEthicWeb.FaucetController do
   end
 
   def create_transfer(conn, %{"address" => address}) do
+    
+    FaucetRateLimiter.register(address, System.monotonic_time())
```

Comment:
Why are you registering the address before to check the rate limiter policy ?
Because I could easily make bomb requests and the genserver will have a huge message queue and blow over.
I think you should check before to write.

Reply:
Registering transactions to monitor has been moved after checking the rate limit